mon-pix/app/styles/components/_training-card.scss

```
@@ -24,10 +24,14 @@
 }
 
 .training-card-content__infos {
-  margin: 0 0 48px 16px;
+  padding: 0 16px 48px;
```

On a accès aux design tokens, donc privilégier : 

```suggestion
  padding: 0 $spacing-s $spacing-xxl;
```